src/taretto/application_skel/{{cookiecutter.application_name}}/{{cookiecutter.application_name}}/base/application/__init__.py

```diff
@@ -0,0 +1,43 @@
+import importscan
+
+from pkg_resources import iter_entry_points
+from taretto.modeling import EntityCollections
+from taretto.context import ImplementationContext
+
+from {{cookiecutter.application_name}}.base.application.implementations.web_ui import ViaWebUI
+
+
+class Application(object):
```

Comment:
not specific to this pr, but is there a reason why we're hanging on to py2 support?